src/app/project/directives/obs-benthic-lit/obs-benthic-lit-list.dir.js

```diff
@@ -60,6 +60,9 @@ angular.module('app.project').directive('obsBenthicLitList', [
         loadBenthicAttributesLookup();
 
         scope.getBenthicAttributes = function() {
+          if (_.isFunction(scope.benthicAttributeChoices)) {
```

Comment:
I'm sure these `scope....AttributeChoices` vars are right, but I only see e.g. `$scope.benthicAttributes` getting set in the controllers, so don't understand where this is coming from.

Reply:
module.js `resolve` -> `collect-benthic-lit-transect-method0-ctrl.js` -> `benthicpitprotocol.observations.form.tpl.html` ->  `obs-benthic-lit-list.dir.js`